build_tools/scripts/integrate/iree_utils.py

```diff
@@ -75,7 +75,7 @@ def git_is_porcelain(*, repo_dir=None):
 
 def git_check_porcelain(*, repo_dir=None):
     output = git_exec(
-        ["status", "--porcelain", "--untracked-files=no"],
+        ["status", "--porcelain"],
```

Comment:
Can you elaborate on this change? What git state did you have where you wanted to bypass the untracked files check?

Reply:
Ahhh, gotcha.